libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -513,8 +513,9 @@ - (void)authClient:(SFSDKOAuthClient *)client willBeginBrowserAuthentication:(SF
 
 - (BOOL)authClientDidCancelBrowserFlow:(SFSDKOAuthClient *)client {
     BOOL result = NO;
+    SFOAuthInfo *authInfo = [[SFOAuthInfo alloc] initWithAuthType:SFOAuthTypeAdvancedBrowser];
     NSDictionary *userInfo = @{ kSFNotificationUserInfoCredentialsKey: client.credentials,
-                                kSFNotificationUserInfoAuthTypeKey: client.context.authInfo };
+                                kSFNotificationUserInfoAuthTypeKey: authInfo };
```

Comment:
Same question as the other PR -> Why doesn't `client.context.authInfo` have the correct `authType` here? @trooper2013 @paultiarks

Reply:
@bhariharan There is no harm in doing this here. We changed advanced auth to be configured thru a domain/org setting only. The Auth Client is configured to start with UserAgent and kicked off.

Reply:
Gotcha. So the `client` changed dynamically.